### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by mattprice-28**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-spring.datasource.password=${DATASOURCE_PASSWORD}
+
+# For SQL Server
+#spring.datasource.driver-class-name=com.microsoft.sqlserver.jdbc.SQLServerDriver
+#spring.datasource.url=jdbc:sqlserver://${DB_SERVER};databaseName=${DB_NAME}
+#spring.datasource.username=${DATASOURCE_USERNAME}
+#spring.datasource.password=${DATASOURCE_PASSWORD}
+
+# For Postgres
+spring.datasource.url=jdbc:postgresql://${AZ_COLLECTION_DB_SERVER}:5432/${AZ_COLLECTION_DB_NAME}?ssl=true&sslmode=prefer
```
[the remark this answers is not in the document]
Comment (mattprice-28) [reply src/main/resources/application.properties]: i've commented it out for now

File: Dockerfile [hunk 2]
```diff
@@ -1,9 +1,17 @@
 FROM maven:3.6.0-jdk-12-alpine as first_layer
 ENV DB_NAME=CollectionDev
-ENV DB_SERVER=sql_server.TakeOnContainerNet
-ENV DATASOURCE_USERNAME=sa
-ENV DATASOURCE_PASSWORD=aVerySecurePassword!123
+# ENV DB_SERVER=sql_server.TakeOnContainerNet
+# ENV DB_SERVER=sql-server-db-chart
+ENV DB_SERVER=db-layer-takeon
```
Comment (RyanAldy) [inline Dockerfile]: These are out of date so this Dockerfile needs tidying up.  We now use configmaps and secrets to store this information

File: persistence/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
```
Comment (RyanAldy) [inline persistence/templates/NOTES.txt]: I don't think we should be including Helm Charts for all containers in this repository.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
-spring.datasource.password=${DATASOURCE_PASSWORD}
+
+# For SQL Server
+#spring.datasource.driver-class-name=com.microsoft.sqlserver.jdbc.SQLServerDriver
+#spring.datasource.url=jdbc:sqlserver://${DB_SERVER};databaseName=${DB_NAME}
+#spring.datasource.username=${DATASOURCE_USERNAME}
+#spring.datasource.password=${DATASOURCE_PASSWORD}
+
+# For Postgres
+spring.datasource.url=jdbc:postgresql://${AZ_COLLECTION_DB_SERVER}:5432/${AZ_COLLECTION_DB_NAME} # ?ssl=true&sslmode=prefer
```
Comment (RyanAldy) [inline src/main/resources/application.properties]: Should these still be called "AZ_*****" as this was for Azure.  DB_SERVER, DB_USERNAME, DB_PASSWORD would be better

**Pull request merged**